man/mkdocs/grassdocs.css

```diff
@@ -1,10 +1,129 @@
 :root > * {
-  --md-primary-fg-color:        #088B36;
-  --md-primary-fg-color--light: #088B36;
-  --md-primary-fg-color--dark:  #088B36;
-  --md-footer-bg-color:         #088B36;
-  --md-footer-bg-color--light:  #088B36;
-  --md-footer-bg-color--dark:   #088B36;
+
+  /* GRASS Variables */
+  /* ----------------------------------------------------------------------------- */
+  --primary-color: rgba(76, 176, 91, 1); /* Dark green */
+  --primary-color--light: rgba(158, 210, 156, 1); /* Light green */
```

Comment:
Two dashes, intentional? (I see it in the original code.)

Reply:
Yes. The CSS convention for custom properties begins with `--` (e.g.,  `--gs-primary-color`). Material MkDocs uses the double `--` (e.g.,  `--gs-primary-color--light`) to organize variant custom properties.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascading_variables/Using_CSS_custom_properties

Reply:
Thanks for the explanation. The Material MkDocs practice is what I was after.